src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2315,6 +2325,27 @@ public static function registerRateLimiter(ContainerBuilder $container, string $
         $container->registerAliasForArgument($limiterId, RateLimiterFactory::class, $name.'.limiter');
     }
 
+    private function registerUidConfiguration(array $config, ContainerBuilder $container, PhpFileLoader $loader)
+    {
+        $loader->load('uid.php');
+
+        $container->getDefinition('uuid.factory')
+            ->setArguments([
+                $config['default_uuid_version'],
+                $config['time_based_uuid_version'],
+                $config['name_based_uuid_version'],
+                UuidV4::class,
```

Comment:
Why don't we make this one configurable as well? :thinking:

Reply:
We allow configuring where an alternative is possible, but random-based UUID allow only v4, there is nothing to configure here, since we deal only with numbers in the config, not classes.